test/python/pulse/test_schedule.py

```diff
@@ -674,22 +675,31 @@ def test_filter_inst_types(self):
         for _, inst in no_pulse_and_fc.instructions:
             self.assertFalse(isinstance(inst, (Play, ShiftPhase)))
         self.assertEqual(len(only_pulse_and_fc.instructions), 4)
-        self.assertEqual(len(no_pulse_and_fc.instructions), 3)
+        self.assertEqual(len(no_pulse_and_fc.instructions), 4)
 
         # test on ShiftPhase
         only_fc, no_fc = \
             self._filter_and_test_consistency(sched, instruction_types={ShiftPhase})
         self.assertEqual(len(only_fc.instructions), 1)
-        self.assertEqual(len(no_fc.instructions), 6)
+        self.assertEqual(len(no_fc.instructions), 7)
 
         # test on SetFrequency
-        only_sf, no_sf = \
+        only_setf, no_setf = \
```

Comment:
```suggestion
        only_setf, no_setf =self._filter_and_test_consistency(
            sched, instruction_types=[SetFrequency]))
```

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/4390/commits/7ecf5ab6be4f22c58b889b1969bfba31b34a4545.